src/client.rs

```diff
@@ -50,15 +51,21 @@ impl IggyClient {
     /// This initializes a new runtime for asynchronous operations.
     /// Future versions might utilize asyncio for more Pythonic async.
     #[new]
-    fn new() -> Self {
+	#[pyo3(signature = (conn=None))]
+    fn new(conn: Option<String>) -> Self {
         // TODO: use asyncio
         let runtime = Builder::new_multi_thread()
             .worker_threads(4) // number of worker threads
             .enable_all() // enables all available Tokio features
             .build()
             .unwrap();
+    let client = IggyClientBuilder::new()
+        .with_tcp()
+        .with_server_address(conn.unwrap_or("127.0.0.1:8090".to_string()))
+        .build()
+		.unwrap();
```

Comment:
I would recommend using this:
```
IggyClient::from_connection_string(&iggy_address)
```

Iggy has its own connection string like `iggy://iggy:iggy@127.0.0.1:8090`.

Reply:
The example using TCP is expecting the raw hostname + port, while the `IggyClient` (which is an additional wrapper on top of the selected transport layer) supports the custom connection string.